package io.nextweb.promise.exceptions;

public interface ExceptionListener {

	public void onFailure(ExceptionResult r);

}
